App.js

```diff
@@ -6,38 +6,488 @@ import styles from './src/style';
 import RouteSelection from './src/components/RouteSelection';
 
 const url = 'https://sase-labs-2020.github.io/assets/directions/coffman_yudof.json'
+import MapView, {Marker} from 'react-native-maps';
 
-export default class App extends Component {
```

Comment:
Please integrate the content from `GraphEdge.js` and `BuildingMarker.js` into a file called `Map.js`. `App.js` should only hold the navigation bar component.